xcmailr-webapp/pom.xml

```diff
@@ -157,51 +168,41 @@
             </resource>
         </resources>
```

Comment:
I would like to keep the `version.ftl.html file` together with all the other template files so there will be no confusion where this file is located. The following `resources` definition makes this possible.

```
        <resources>
            <resource>
                <directory>src/main/java</directory>
                <excludes>
                    <exclude>**/*.java</exclude>
                    <exclude>views/layout/version.ftl.html</exclude>
                </excludes>
            </resource>
            <resource>
                <directory>src/main/java</directory>
                <filtering>true</filtering>
                <includes>
                    <include>views/layout/version.ftl.html</include>
                </includes>
            </resource>
            <resource>
                <directory>src/main/resources</directory>
            </resource>
        </resources>
```

WDYT? Would of course require the file to be moved from `src\main\resources` to `src\main\java\views\layout`.

Reply:
Moved all resource files to 'src/main/resources' and 'src/test/resources' respectively following Maven conventions.

Reply:
All right. In this case we can also get rid of this block:

```
            <resource>
                <directory>src/main/java</directory>
                <excludes>
                    <exclude>**/*.java</exclude>
                </excludes>
            </resource>
```

Reply:
Narf... Nice catch! Done.